src/stylesheets/blocks/_concept.scss

```diff
@@ -30,36 +41,238 @@
 
   .lg-text {
     font-size: rem(30);
-    margin-bottom: rem(25);
+    margin-bottom: 0;
+
+    @media (screen(lg)) {
+      font-size: rem(60);
+    }
   }
 
   .sm-sans-serif-text {
-    padding-bottom: rem(25);
+    padding-top: rem(25);
+
+    @media (screen(lg)) {
+      text-align: left;
+    }
   }
 }
 
 .concept-section__list {
   list-style: none;
   padding: 0;
+  margin: 0;
+
+  :before {
```

Comment:
This is actually applying to ALL `:before` instances in the list. Since you're targeting the list item pseudo element, it should be nested under `.concept-section__list__item` for clarity. Same for the `:after` block below.

Reply:
I'm probably doing something wrong, but they all disappear when I nest them under `.concept-section__list__item`

Reply:
Did you add the `&` in front of the pseudo selector?

```
.concept-section__list__item {
  &::before {
    ...
  }
}
```

That will ensure the resulting selector looks like this: `.concept-section__list__item:before`.